geode-core/src/main/java/org/apache/geode/internal/cache/DiskStoreFactoryImpl.java

```diff
@@ -144,7 +144,10 @@ public DiskStore create(String name) {
     // As a simple fix for 41290, only allow one DiskStore to be created
     // at a time per cache by syncing on the cache.
```

Comment:
This comment is no longer entirely accurate.

Reply:
Agree.